markets/perps-market/contracts/storage/AsyncOrder.sol

```diff
@@ -540,8 +547,9 @@ library AsyncOrder {
         // do same thing for liquidation rewards and account for minimum liquidation margin
         uint256 requiredLiquidationRewardMargin = GlobalPerpsMarketConfiguration
             .load()
-            .minimumLiquidationReward(
-                currentTotalLiquidationRewards + newLiquidationReward - oldLiquidationReward
+            .minimumKeeperReward(
+                currentTotalLiquidationRewards + newLiquidationReward - oldLiquidationReward,
```

Comment:
the `costOfExecution` here is based on the old position and needs to be recalculated for new position. and added to this line

```
uint256 newTotalLiquidationRewards = currentTotalLiquidationRewards + newLiquidationReward - oldLiquidationReward + newCostOfExecution;
uint256 requiredLiquidationRewardMargin = GlobalPerpsMarketConfiguration
  .load().minimumKeeperReward(newTotalLiquidationRewards, newCostOfExecution);
```